### PR TITLE
Improve inline documentation for CLI

### PR DESCRIPTION
**Pull request by bszwarc**
**Description**

Changes proposed in this pull request:

- Improve inline documentation for commands
- Improve error messages

**Related issue(s)**
See also #177

### CODE REVIEW

File: internal/kube/client.go [hunk 1]
```diff
@@ -82,7 +82,7 @@ func (c *client) Octopus() octopus.OctopusInterface {
 func (c *client) IsPodDeployed(namespace, name string) (bool, error) {
 	_, err := c.Static().CoreV1().Pods(namespace).Get(name, metav1.GetOptions{IncludeUninitialized: true})
 	if err != nil {
-		if strings.Contains(err.Error(), "not found") {
+		if strings.Contains(err.Error(), "Not found") {
```
Comment (igor-karpukhin) [inline internal/kube/client.go]: That won't work. The error messages coming from a kubernetes API are in lower case. Either leave it in lower case or all checks should be wrapped with `strings.ToLower()`

File: internal/kube/client.go [hunk 2]
```diff
@@ -104,7 +104,7 @@ func (c *client) IsPodDeployedByLabel(namespace, labelName, labelValue string) (
 func (c *client) WaitPodStatus(namespace, name string, status corev1.PodPhase) error {
 	for {
 		pod, err := c.Static().CoreV1().Pods(namespace).Get(name, metav1.GetOptions{IncludeUninitialized: true})
-		if err != nil && !strings.Contains(err.Error(), "not found") {
+		if err != nil && !strings.Contains(err.Error(), "Not found") {
```
Comment (igor-karpukhin) [inline internal/kube/client.go]: Same thing here

File: pkg/kyma/cmd/install/cmd.go [hunk 3]
```diff
@@ -830,16 +830,16 @@ func (cmd *command) releaseFile(path string) string {
...
 		return nil
 	}
 	minikubeIP = strings.TrimSpace(minikubeIP)
 
 	for k, v := range patchMap {
 		for _, pData := range v {
 			if _, err := cmd.Kubectl().RunCmd("-n", "kyma-installer", "get", k); err != nil {
-				if strings.Contains(err.Error(), "not found") {
-					cmd.CurrentStep.LogInfof("resource '%s' not found, won't be patched", k)
+				if strings.Contains(err.Error(), "Not found") {
```
Comment (igor-karpukhin) [inline pkg/kyma/cmd/install/cmd.go]: Here is the same thing

File: internal/kube/client.go (hunk 1 above)
Comment (bszwarc) [reply internal/kube/client.go]: got it - I will change it :)

File: pkg/kyma/cmd/install/cmd.go (hunk 3 above)
Comment (bszwarc) [reply pkg/kyma/cmd/install/cmd.go]: Will change :)

File: pkg/kyma/cmd/uninstall/cmd.go [hunk 4]
```diff
@@ -182,17 +182,17 @@ func (cmd *command) deleteTiller() error {
 	}
 
 	err = cmd.K8s.Static().RbacV1().RoleBindings("kube-system").Delete("tiller-certs", &metav1.DeleteOptions{})
-	if err != nil && !strings.Contains(err.Error(), "not found") {
+	if err != nil && !strings.Contains(err.Error(), "Not found") {
```
Comment (igor-karpukhin) [inline pkg/kyma/cmd/uninstall/cmd.go]: Same thing here as above

File: pkg/kyma/cmd/uninstall/cmd.go [hunk 5]
```diff
@@ -182,17 +182,17 @@ func (cmd *command) deleteTiller() error {
...
 		return err
 	}
 
 	err = cmd.K8s.Static().RbacV1().Roles("kube-system").Delete("tiller-certs-installer", &metav1.DeleteOptions{})
-	if err != nil && !strings.Contains(err.Error(), "not found") {
+	if err != nil && !strings.Contains(err.Error(), "Not found") {
```
Comment (igor-karpukhin) [inline pkg/kyma/cmd/uninstall/cmd.go]: Same thing here as above

File: pkg/kyma/cmd/uninstall/cmd.go [hunk 6]
```diff
@@ -182,17 +182,17 @@ func (cmd *command) deleteTiller() error {
...
 		return err
 	}
 
 	err = cmd.K8s.Static().CoreV1().ServiceAccounts("kube-system").Delete("tiller-certs-sa", &metav1.DeleteOptions{})
-	if err != nil && !strings.Contains(err.Error(), "not found") {
+	if err != nil && !strings.Contains(err.Error(), "Not found") {
```
Comment (igor-karpukhin) [inline pkg/kyma/cmd/uninstall/cmd.go]: Same thing here as above

File: internal/kyma_test.go [hunk 7]
```diff
@@ -17,26 +17,26 @@ func TestRunCmd(t *testing.T) {
...
 		},
 		{
 			name:           "Incorrect command",
-			description:    "test a command that exits with error",
-			cmd:            []string{"ehco", "this is wrongly spelled"},
+			description:    "Checks if a command is correct. If not, exists and returns an error.",
+			cmd:            []string{"ehco", "This is spelled incorrectly"},
 			expectedOutput: "",
-			expectedErr:    errors.New("Failed executing command 'ehco [this is wrongly spelled]' with output '' and error message 'exec: \"ehco\": executable file not found in $PATH'"),
+			expectedErr:    errors.New("Executing command 'ehco [this is wrongly spelled]' failed with output '' and error message 'exec: \"ehco\": executable file not found in $PATH'"),
```
Comment (igor-karpukhin) [inline internal/kyma_test.go]: `"this is wrongly spelled"` should be also replaced in the `expectedErr` field within the `[]`

Comment (bszwarc) [reply internal/kyma_test.go]: Done.

**Pull request merged**